### ISSUE TITLE
Scale hyperparameter is different between PastEncoder and FutureEncoder.

### ISSUE THREAD

**Issue by SeongjuLee** [MediaBrain-SJTU/GroupNet#6]
In `PastEncoder`, `MS_HGNN_hyper` layers takes `scale` arguments and their values are taken from `args.hyper_scales`.
However, in `FutureEncoder`, `MS_HGNN_hyper` layers takes `scale` as the values of 1, 2, and 3.
What is the difference?

**Comment by sjtuxcx** [comment]
Thanks for your question. Sorry it is a mistake since we treat the default scales as [5,11,3]. Both of PastEncoder and FutureEncoder take the scale from args.hyper_scales. We have modified the code.

**Comment by SeongjuLee** [comment]
Thanks for your replying. I checked the scale parameter of `FutureEncoder`.
I have one more question.
The performance of pretrained model you uploaded is different with the values in the paper.
Would you tell the what/how different?

**Comment by sjtuxcx** [comment]
This is because we modify the loss function thanks to the suggestion in  #2 . The performance further improves.

**Comment by SeongjuLee** [comment]
Thanks.
